UI/src/application.js

```diff
@@ -427,7 +445,7 @@ var URLRouter = function(map, backend, models) {
         multiElementEventCategory: asset.multiElementEventCategory,
         singleElementEventCategory: asset.singleElementEventCategory,
         style: PiecewiseLinearAssetStyle(applicationModel),
-        formElements: PiecewiseLinearAssetFormElements(asset.unit, asset.editControlLabels, asset.className, asset.defaultValue)
```

Comment:
Maybe we could still use the old name PiecewiseLinearAssetFormElements, as the client does not know that internally there is some differentiation based on the element type